roles/iridl/tasks/CentOS7_setup.yaml

```diff
@@ -0,0 +1,8 @@
+- name: Fix CentOS7 Repositories
+  include_tasks: CentOS7_vault.yaml
+
+- name: Upgrade CA Certificates
+  package:
+    name:
+      - ca-certificates
```

Comment:
laceStill not pinned.

Is this necessary at all? It's already installed in a couple other places.

Reply:
I can't believe this!  I was so sure I did it.  

You're right.  I should really only do it here and remove it from tasks/main.yaml which calls this routine.  Because we probably don't want to pin the same version for centos7 and centos9, I will create a Cento9_setup.yaml file and install the ca_certificates package there?

Can I assume the miniconda role will always be called after iridl role and not upload it there too?